lib/netpacket/netpacket_write.c

```diff
@@ -80,7 +80,7 @@ netpacket_write_file(NETPACKET_CONNECTION * NPC,
 	packetbuf[8] = class;
 	memcpy(&packetbuf[9], name, 32);
 	memcpy(&packetbuf[41], nonce, 32);
-	be32enc(&packetbuf[73], buflen);
+	be32enc(&packetbuf[73], (uint32_t)buflen);
```

Comment:
Would you rather that this was

    buflen & 0xffffffff

instead?  There's a whole bunch of `be32enc()` and `le32enc()` places that currently use implicit casting.